cmd/armadactl/README.md

```diff
@@ -0,0 +1,111 @@
+# Armadactl
+
+Armadactl is a command-line tool used for managing jobs in the Armada workload orchestration system. It provides functionality for creating, updating, and deleting jobs, as well as monitoring job status and resource usage.
+
+## Pre-requisites
+- Git
+- Docker
+- Helm v3.5+
+- Kind v0.11.1+
+- Kubectl
+
+## Installation
```

Comment:
Can you remove this installation section?

If you want to add a README.md about `armadactl` and this cmd I think that is fine.  

But I would ask that we don't replicate text that is somewhere else in the repo.  It is hard to keep docs up to date.

Reply:
I guess you are right... I am removing the whole installation section...